mapflow/mapflow.py

```diff
@@ -1503,26 +1503,25 @@ def post_processing_callback(self, _: QNetworkReply, processing_name: str) -> No
         )
```

Comment:
Why do we request /projects/ here, not /user/status?

Also, we should better do this in get_processings_callback, so that it would update regularly, not only on POST processing